esp/logging/logginglib/logthread.cpp

```diff
@@ -92,6 +92,9 @@ CLogThread::CLogThread(IPropertyTree* _cfg , const char* _service, const char* _
 
         logFailSafe.setown(createFailSafeLogger(_service, _agentName, logsDir));
     }
+    time_t tNow;
+    time(&tNow);
+    localtime_r(&tNow, &m_startTime);
```

Comment:
@afishbeck Those 3 lines were removed from old logging client by mistake. If m_startTime is not initialized, newly created tank file (even if it contains unfinished job from previous runs of the ESP) will be Rollovered.